contrib/native/client/src/clientlib/drillClientImpl.cpp

```diff
@@ -1722,8 +1774,68 @@ void DrillClientImpl::shutdownSocket(){
     DRILL_MT_LOG(DRILL_LOG(LOG_TRACE) << "Socket shutdown" << std::endl;)
 }
 
+namespace { // anonymous
```

Comment:
no, it's an anonymous namespace: the actual name is generated at compilation time, and it prevents symbols to leak from outside object file, or clash with other symbols (if not declared static).